src/scalper.ts

```diff
@@ -101,62 +99,74 @@ class Scalper {
     }
 
     console.log(this.symbol, 'Choosing market to trade');
-    this.perpMarketConfig = getMarketByBaseSymbolAndKind(
-      this.groupConfig,
-      this.symbol,
-      'perp',
-    );
-    this.marketIndex = this.perpMarketConfig.marketIndex;
-    if (this.marketIndex === undefined) {
-      console.log('No Mango Market Exists. Run OpenBook');
-      await this.scalperOpenBook(dipProduct);
-      return;
-    }
 
-    const mangoGroup: MangoGroup = await this.client.getMangoGroup(
-      this.groupConfig.publicKey,
-    );
-    const perpMarket: PerpMarket = await mangoGroup.loadPerpMarket(
-      this.connection,
-      this.marketIndex,
-      this.perpMarketConfig.baseDecimals,
-      this.perpMarketConfig.quoteDecimals,
-    );
+    if (this.mode === ScalperMode.Perp) {
+      // Load Mango
+      // TODO: Add Priority Fee
+      this.mangoClient = MangoClient.connect(
+        this.provider,
+        cluster,
+        MANGO_V4_ID[cluster],
+        {
+          idsSource: 'get-program-accounts',
+        },
+      );
+      const mangoAccount = await this.mangoClient.getMangoAccount(
```

Comment:
why cant we store this on the scalper object like all similar objects?

Reply:
We can. Wasn't sure if its best to keep that object small, seems like it keeps growing and growing